tests/test_sr.py

```diff
@@ -377,6 +391,8 @@ def test_datetime_item_construction_from_datetime(self):
         assert i.ValueType == 'DATETIME'
         assert i.ConceptNameCodeSequence[0] == name
         assert i.DateTime == DT(value)
+        assert i.name == CodedConcept(*name)
+        assert i.value == value
```

Comment:
This will need a bit of work. We currently use `unittest.TestCase`, which is incompatible with `pytest.fixture` (see [docs](https://docs.pytest.org/en/6.2.x/unittest.html#pytest-features-in-unittest-testcase-subclasses))

Reply:
This particular class wouldn't need to be subclassed from `unittest.TestCase`, but we should then change this across the board.

Reply:
Hmm... I see. Well we already have several tests that do not use `unittest.TestCase`, for example those in `test_valuerep` and `test_utils`, so although it's a bit messy, it doesn't seem like a little more mix and match would hurt too much.

I've just been bitten too many times before by this `datetime_conversion` thing and really think having the test would be a good idea...

Reply:
Ok. Maybe we should just rip out `unittest.TestCase`. This caused problems before.

Reply:
Perhaps, but I don't see any urgency.